src/components/CardData/CardData.js

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
-import './CardData.css';
+import React from 'react'
+import './CardData.css'
 
-const CardData = () => {
-  return (
-    <div className='card-data'>
+const CardData = ()=>{
+  return(
+    <div>
```

Comment:
This break tests because the className is required as part of the test. It's also needed for CSS. The removal of the semi-colons are also kicking back ESLint errors.